.github/workflows/chromatic.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       - name: Publish to Chromatic - Components
         uses: chromaui/action@latest
         with:
+          autoAcceptChanges: "main"
```

Comment:
The quotes aren't necessary in yaml, right?

Reply:
This is how it was written in the documentation: https://www.chromatic.com/docs/github-actions/